magento/magento2
# Fixed : Getting error after dismiss admin notification

### Description (*)
    This Pull Request fix error when if we dismiss any notification in admin panel

### Related Pull Requests


### Fixed Issues (if relevant)
1. Fixes magento/magento2#36581

### Manual testing scenarios (*)

1. Login Admin
2. Go to Catalog > Products
3. Choose mass action Update attributes
4. Update the Product Name and click Save
5. At this point, new message will be added to top.
6. Click X to dismiss the message > No error should occur

### Questions or comments


### Contribution checklist (*)
